### ISSUE TITLE
[Bug]: PPO fails to load previous model from zip file

### ISSUE THREAD

**Issue by Soumil32** [DLR-RM/stable-baselines3#1131]
### 🐛 Bug

PPO.load() produces an assertion error when loading my model.

Removing `assert callable(value_schedule)` (line which is generating the problem) in commom/utils.py line 91 fixes the error and causes no further problems. The model gets loaded normally and runs as expected. 

This problem occurs no matter where I am executing the code. It happens on my local machine, google colab and paperspace (which just like google colab).

### To Reproduce

```python
from stable_baselines3 import PPO
PPO.load('path/to/model')
```

### Relevant log output / Error message

```shell
---------------------------------------------------------------------------
AssertionError                            Traceback (most recent call last)
Cell In [9], line 2
      1 # Load model
----> 2 model = PPO.load('./train/best_model_500000')

File c:\Users\soumi\Documents\Mario ai\venv\lib\site-packages\stable_baselines3\common\base_class.py:806, in BaseAlgorithm.load(cls, path, env, device, custom_objects, print_system_info, force_reset, **kwargs)
    804 model.__dict__.update(data)
    805 model.__dict__.update(kwargs)
--> 806 model._setup_model()
    808 # put state_dicts back in place
    809 model.set_parameters(params, exact_match=True, device=device)

File c:\Users\soumi\Documents\Mario ai\venv\lib\site-packages\stable_baselines3\ppo\ppo.py:173, in PPO._setup_model(self)
    170 super()._setup_model()
    172 # Initialize schedules for policy/value clipping
--> 173 self.clip_range = get_schedule_fn(self.clip_range)
    174 if self.clip_range_vf is not None:
    175     if isinstance(self.clip_range_vf, (float, int)):

File c:\Users\soumi\Documents\Mario ai\venv\lib\site-packages\stable_baselines3\common\utils.py:91, in get_schedule_fn(value_schedule)
     89     value_schedule = constant_fn(float(value_schedule))
     90 else:
---> 91     assert callable(value_schedule)
     92 return value_schedule

AssertionError:
```

### System Info

OS: Windows-10-10.0.19044-SP0 10.0.19044
Python: 3.8.10
Stable-Baselines3: 1.6.2
PyTorch: 1.12.1+cpu
GPU Enabled: False
Numpy: 1.23.4
Gym: 0.21.0

### Checklist

- [X] I have checked that there is no similar [issue](https://github.com/DLR-RM/stable-baselines3/issues) in the repo
- [X] I have read the [documentation](https://stable-baselines3.readthedocs.io/en/master/)
- [X] I have provided a minimal working example to reproduce the bug
- [x] I've used the [markdown code blocks](https://help.github.com/en/articles/creating-and-highlighting-code-blocks) for both code and stack traces.

**Comment by qgallouedec** [comment]
This issue has been fixed with https://github.com/DLR-RM/stable-baselines3/pull/1123 and will be part of v1.7.0
Should be solved if you [install the bleeding-edge-version of stable-baselines3](https://stable-baselines3.readthedocs.io/en/master/guide/install.html#bleeding-edge-version)

**Comment by Soumil32** [comment]
I am using version 1.6.2 which is the latest stable version

**Comment by Soumil32** [comment]
> This issue has been fixed with #1123 and will be part of v1.7.0 Should be solved if you [install the bleeding-edge-version of stable-baselines3](https://stable-baselines3.readthedocs.io/en/master/guide/install.html#bleeding-edge-version)

Thankyou that worked!
